client-apis/encrypt.md

```diff
@@ -53,6 +54,10 @@ A CMM that implements the [CMM interface](#cmm-interface.md).
 
 A Keyring that implements the [keyring interface](#keyring-interface.md).
 
+### Algorithm Suite
+
+The [algorithm suite](#algorithm-suite.md) that SHOULD be used for encryption.
```

Comment:
If I understand correctly. The input Algorithm Suite has no direct relation to the algorithm suite used for Encryption (or signing). This like should probably read: 
```
The [algorithm suite](#algorithm-suite.md) that MUST be passed to the CMM.
```

or something along those lines.

Reply:
It's hard to know where to draw this line, but I'm trying to define this for the user's standpoint. As far as someone interacting with the Encrypt API is concerned, unless the CMM being used thinks it knows better (remember that CMMs SHOULD return encryption materials with the same algorithmSuiteID as was in the GetEncryptionMaterials call), the algorithm suite ID included as part of the encrypt call SHOULD just be the algorithm suite that is used for the encryption process.

Someone can read further into the spec to see exactly how this input is used such that the concept defined above is true.

Reply:
Sounds good to me!